src/fhir/sdc-import-common.js

```diff
@@ -885,7 +885,24 @@ function addCommonSDCImportFns(ns) {
    * @param qr a QuestionnaireResponse instance
    * @returns {{}} an updated LForms form definition, with answer data
    */
-  qrImport.mergeQuestionnaireResponseToLForms = function(formData, qr) {
+  let mismatchItem = [];
```

Comment:
I think this should be declared inside function scope. It's not used outside the function, right?